core/src/main/scala/com/wix/accord/combinators/StringCombinators.scala

```diff
@@ -31,6 +31,14 @@ trait StringCombinators {
   class EndsWith( suffix: String )
     extends NullSafeValidator[ String ]( _ endsWith suffix, _ -> s"must end with '$suffix'" )
 
+  /** A validator that succeeds only if the provided string is not blank. */
+  class NotBlank
+    extends NullSafeValidator[ String ]( !_.trim.isEmpty, _ -> "must not blank" )
```

Comment:
I'm concerned about `java.lang.String.trim` -- it assumes every codepoint under 0x20 (space) is whitespace, but ignores other Unicode whitespace code points (see [full list](https://en.wikipedia.org/wiki/Whitespace_character#Unicode) on Wikipedia).

I'm inclined to allow it because this is roughly the expected behavior for most developers, but the implementation detail should probably be documented in the ScalaDocs.